lib/components/ConflictPicker.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-	<NcModal class="conflict-picker"
+	<NcDialog class="conflict-picker"
```

Comment:
`NcDialog` **requires** the `name` prop for accessibility reasons (a11y labeling).
So put the `name` into the `name` prop and remove the `h2` from the header below.

```suggestion
	<NcDialog class="conflict-picker"
		:name="name"
```

Reply:
Ahhh, good catch. Added :) The view after the changes:
![image](https://github.com/nextcloud-libraries/nextcloud-upload/assets/110193237/5f9a8f8f-cdfd-4e27-889e-33815ae53096)

Reply:
are you sure this is after the changes? Because I would expect the title to be placed on the top centered.

Reply:
![image](https://github.com/nextcloud-libraries/nextcloud-upload/assets/110193237/4d0bb719-8061-45b0-935e-c75386eb8a4a)
You where right 😅 New screenshot here ^